.github/workflows/api_doc_build.yml

```diff
@@ -15,13 +15,23 @@ jobs:
       - uses: actions/checkout@v4
         with:
           ref: bagatur/api_docs_build
+          path: main
```

Comment:
sets default repo working directory to ./main

Reply:
maybe just call this langchain? and google langchain-google? ie the repo names